components/ProtegeProgramCard.tsx

```diff
@@ -0,0 +1,44 @@
+import {Box, Stack, Typography} from '@mui/joy';
+import Card from '@mui/joy/Card';
+import OutlineButton from '@/components/OutlineButton';
+import Image from 'next/image';
+
+export default function ProtegeProgramCard({ icon, text, type, description }: any) {
+  return (
+    <Card
+      variant='outlined'
+      className='border-round'
+      sx={{
+        width: {xs: '300px', lg: '350px'},
+        height: 'auto',
+        border: '2px solid #93AED3'
+      }}
+    >
+      <Box sx={{
+          position: 'absolute',
+          top: '-50px',
+          left: '50%',
+          transform: 'translateX(-50%)',
+          width: {xs: '90px', sm: '100px'},
+          height: {xs: '90px', sm: '100px'},
+          borderRadius: '50%',
+          border: '2px solid #93AED3',
+          backgroundColor: 'white',
+          display: 'flex',
+          alignItems: 'center',
+          justifyContent: 'center',
+      }}>
+        <Image src={icon} alt={text} width={70} height={70}/>
+      </Box>
+      <Stack direction="column" spacing={5} alignItems='center' pt={8} pb={4}>
+        <Typography level='subtitle-lg'>
+          {type}
+        </Typography>
+        <Typography level='body' textAlign='center' px={{xs: 2, sm: 3, md: 4, lg: 5}}>
+          {description}
+        </Typography>
+        <OutlineButton text='Apply now' link='sth'/>
```

Comment:
For the OutlineButton component, can you make the hover effect darker, like it is in the figma?

<img width="425" alt="Screen Shot 2024-07-25 at 4 35 04 pm" src="https://github.com/user-attachments/assets/5809ab67-b1b4-47e7-8e2f-a31f116c37c8">